api/src/com/cloud/deploy/DeploymentPlanner.java

```diff
@@ -1,323 +1,306 @@
-            }
-            _hostIds.add(hostId);
-        }
-
-        public void addHostList(Collection<Long> hostList) {
-            if (_hostIds == null) {
-                _hostIds = new HashSet<Long>();
-            }
-            _hostIds.addAll(hostList);
-        }
-
-        public boolean shouldAvoid(Host host) {
-            if (_dcIds != null && _dcIds.contains(host.getDataCenterId())) {
-                return true;
-            }
-
-            if (_podIds != null && _podIds.contains(host.getPodId())) {
-                return true;
-            }
-
-            if (_clusterIds != null && _clusterIds.contains(host.getClusterId())) {
-                return true;
-            }
-
-            if (_hostIds != null && _hostIds.contains(host.getId())) {
-                return true;
-            }
-
-            return false;
-        }
-
-        public boolean shouldAvoid(Cluster cluster) {
-            if (_dcIds != null && _dcIds.contains(cluster.getDataCenterId())) {
-                return true;
-            }
-
-            if (_podIds != null && _podIds.contains(cluster.getPodId())) {
-                return true;
-            }
-
-            if (_clusterIds != null && _clusterIds.contains(cluster.getId())) {
-                return true;
-            }
-            return false;
-        }
-
-        public boolean shouldAvoid(Pod pod) {
-            if (_dcIds != null && _dcIds.contains(pod.getDataCenterId())) {
-                return true;
-            }
-
-            if (_podIds != null && _podIds.contains(pod.getId())) {
-                return true;
-            }
-
-            return false;
-        }
-
-        public boolean shouldAvoid(StoragePool pool) {
-            if (_dcIds != null && _dcIds.contains(pool.getDataCenterId())) {
-                return true;
-            }
-
-            if (_podIds != null && _podIds.contains(pool.getPodId())) {
-                return true;
-            }
-
-            if (_clusterIds != null && _clusterIds.contains(pool.getClusterId())) {
-                return true;
-            }
-
-            if (_poolIds != null && _poolIds.contains(pool.getId())) {
-                return true;
-            }
-
-            return false;
-        }
-
-        public boolean shouldAvoid(DataCenter dc) {
-            if (_dcIds != null && _dcIds.contains(dc.getId())) {
-                return true;
-            }
-            return false;
-        }
-
-        public Set<Long> getDataCentersToAvoid() {
-            return _dcIds;
-        }
-
-        public Set<Long> getPodsToAvoid() {
-            return _podIds;
-        }
-
-        public Set<Long> getClustersToAvoid() {
-            return _clusterIds;
-        }
-
-        public Set<Long> getHostsToAvoid() {
-            return _hostIds;
-        }
-
-        public Set<Long> getPoolsToAvoid() {
-            return _poolIds;
-        }
-    }
-}
+// Licensed to the Apache Software Foundation (ASF) under one

```

Comment:
this class shows way more diffs than needed. please check line endings and force push.